libpod/image/testdata/registries.conf

```diff
@@ -0,0 +1,4 @@
+short-name-mode="enforcing"
+
+[aliases]
+"busybox"="docker.io/library/busybox"
```

Comment:
Is going to respect our proxy over to using quay.io instead of docker for images?

Just checking since the rate-limiting is a very real problem.

Reply:
Very good question! Yes, we will continue pulling from `quay.io`.